examples/with-nonce-manager/src/createNewWallet.ts

```diff
@@ -55,7 +40,7 @@ export async function createNewWallet() {
         `- Wallet ID: ${walletId}`,
         `- Address: ${address}`,
         ``,
-        "Now you can take the address, put it in `.env.local` (`SIGN_WITH=<address>`), then re-run the script.",
+        "Now you can take the address, put it in `.env.local`, then re-run the script.",
```

Comment:
Same as the first file I commented on - this seems intentional and consistent - all good then 👍

Reply:
nice catch here as well, will put it back